lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
-          top: -1px;
-          margin-left: 3px;
-        }
-      }
-
-      td:nth-child(3) {
-        text-align: center;
-      }
-
-      td:last-child {
-        width: 50px;
-        text-align: center;
-      }
-
-      td:nth-child(1) {
-        line-height: 22px;
-        text-align: left;
-        padding-right: $baseline;
-      }
-    }
-
-    .cart-totals {
-      td {
-        &.cart-total-cost {
-          font-size: 1.25em;
-          font-weight: bold;
-          padding: ($baseline/2) 25px;
-        }
-      }
-    }
-  }
-
-  table.order-receipt {
-    width: 100%;
-
-    .order-number {
-      font-weight: bold;
-    }
-    .order-date {
-      text-align: right;
-    }
-    .items-ordered {
-      padding-top: ($baseline*2.5);
-    }
-
-    tr {
-    }
-
-    th {
-      text-align: left;
-      padding: 25px 0 15px 0;
-
-      &.qty {
-        width: 50px;
-      }
-
-      &.u-pr {
-        width: 100px;
-      }
-
-      &.pri {
-        width: 125px;
-      }
-
-      &.curr {
-        width: 75px;
-      }
-    }
-
-    tr.order-item {
-      td {
-        padding-bottom: ($baseline/2);
-
-        span.old-price {
-          text-decoration: line-through !important;
-        }
-      }
-    }
-  }
+  margin-bottom: $baseline;
+  border-bottom: 1px solid $pink;
+  padding: ($baseline/2) 0;
+  background: $card-errors-background-color;
+  color: $pink;
+  font-family: $sans-serif;
 }
 
 #expiry-msg {
-  padding: 15px;
+  @extend %t-icon6;
+  margin-top: ($baseline*0.15);
+  border-top: 1px solid $expiry-border-color;
   background-color: $gray-l5;
-  margin-top: 3px;
-  font-family: $sans-serif;
-  font-size: 14px;
+  padding: ($baseline*0.75);
   text-shadow: 0px 1px 1px $white;
-  border-top: 1px solid #f0f0f0;
+  font-family: $sans-serif;
 }
-.confirm-enrollment {
 
+.confirm-enrollment {
   .title {
-    font-size:24px;
-    border-bottom:1px solid $gray-l5;
-    text-align: left;
-    line-height:70px;
+    @include text-align(left);
+    @include line-height(47.3);
+    @extend %t-icon3;
+    border-bottom: $light-border;
   }
 
   .course-image {
-    float: left;
-    width: 223px;
-    margin-right: ($baseline/2);
+    @include float(left);
+    @include margin-right($baseline/2);
+    width: ($baseline*11.15);
     vertical-align: top;
   }
 
   .enrollment-details {
-    margin-bottom: $baseline;
-    float: left;
-    width: calc(100% - 237px);
+    @include float(left);
+    margin-bottom: ($baseline);
+    width: calc(100% - #{$enrollment-details-width-difference});
 
     .sub-title {
-      font-size: 18px;
+      @extend %t-icon4;
       text-transform: uppercase;
-      color: #9b9b93;
+      color: $enrollment-text-color;
     }
 
     .course-date-label {
-      float: right;
-      color: #9b9b93;
+      @include float(right);
+      color: $enrollment-text-color;
     }
 
     .course-dates {
-      float: right;
-      font-size: 18px;
+      @include float(right);
+      @extend %t-icon4;
```

Comment:
I would use one of the other _typography.scss extends here under the "titles" section instead of using the "icons" sizing extends. (thanks for referencing using _typography.scss though!)